WcaOnRails/app/models/abstract_cached_model.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+class AbstractCachedModel < ActiveRecord::Base
```

Comment:
I came to Ruby from Java and spent a lot of time _terrified_ by the lack of certainty that static type checking used to provide.

Static type checking is also quite constraining. One way to think about it is that now that we (ideally) have test suites, they provide most of that certainty, and so we can get rid of the type checking and take some nice shortcuts in the code. That's my theory on the rise of untyped languages.

To the "Ruby mind", the fact that `include Cachable` would not "compile" if applied to the wrong class makes it safe.